packages/components/organisms/f-footer/test-utils/component-objects/f-footer.component.js

```diff
@@ -14,36 +14,36 @@ module.exports = class Footer extends Page {
         super('organism', 'footer-component');
     }
 
-    get component () { return $(FOOTER_COMPONENT); }
+    static get component () { return $(FOOTER_COMPONENT); }
 
-    get downloadIcons () { return $$(DOWNLOAD_ICONS); }
+    static get downloadIcons () { return $$(DOWNLOAD_ICONS); }
 
-    get socialIcons () { return $$(SOCIAL_ICONS); }
+    static get socialIcons () { return $$(SOCIAL_ICONS); }
 
-    get downloadIcon () { return this.downloadIconValue; }
+    static get downloadIcon () { return this.downloadIconValue; }
 
-    get socialIcon () { return this.socialIconValue; }
+    static get socialIcon () { return this.socialIconValue; }
 
-    get feedbackButton () { return $(FEEDBACK_BUTTON); }
+    static get feedbackButton () { return $(FEEDBACK_BUTTON); }
 
-    get countrySelectorButton () { return $(COUNTRY_SELECTOR_BUTTON); }
+    static get countrySelectorButton () { return $(COUNTRY_SELECTOR_BUTTON); }
 
-    get countries () { return $$(COUNTRY_LIST); }
+    static get countries () { return $$(COUNTRY_LIST); }
 
-    get footerLinkList () { return $(FOOTER_LINK_LIST); }
+    static get footerLinkList () { return $(FOOTER_LINK_LIST); }
 
-    get countryLink () { return this.countryValue != null ? this.countryValue : 'Please set a country value'; }
+    static get countryLink () { return this.countryValue != null ? this.countryValue : 'Please set a country value'; }
 
     set expectedDownloadIcon (icon) {
-        this.downloadIconValue = this.downloadIcons.filter(element => element.getAttribute('data-test-id').includes(icon))[0];
+        [this.downloadIconValue] = this.downloadIcons.filter(element => element.getAttribute('data-test-id').includes(icon));
     }
 
     set expectedSocialIcon (icon) {
-        this.socialIconValue = this.socialIcons.filter(element => element.getAttribute('data-test-id').includes(icon))[0];
+        [this.socialIconValue] = this.socialIcons.filter(element => element.getAttribute('data-test-id').includes(icon));
```

Comment:
I did not know you could assign instance properties inside a destructure like this, nice.